### PR TITLE
SEO-175680-Xamarin-UG-page-with-redirects-link

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: Xamarin/Effects-View/Getting-Started.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ You can add SfEffectsView reference using one of the following methods:
 
 **Method 1: Adding SfEffectsView reference from nuget.org**
 
-Syncfusion Xamarin components are available in [nuget.org](https://www.nuget.org/). To add SfEffectsView to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Xamarin.Core](https://www.nuget.org/packages/Syncfusion.Xamarin.Core), and then install it.
+Syncfusion Xamarin components are available in [nuget.org](https://www.nuget.org). To add SfEffectsView to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Xamarin.Core](https://www.nuget.org/packages/Syncfusion.Xamarin.Core), and then install it.
```
Comment (Yvone-Atieno) [inline Xamarin/Effects-View/Getting-Started.md:28]: Revert this change

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Effects-View/Getting-Started.md (hunk 1 above)
Comment (Edithabich) [reply Xamarin/Effects-View/Getting-Started.md]: reverted.

File: Xamarin/ListView/Grouping.md [hunk 2]
```diff
@@ -616,7 +616,7 @@ The [SfListView.GroupCollapsing](https://help.syncfusion.com/cr/xamarin/Syncfusi
 The [GroupExpandCollapseChangingEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.ListView.XForms.GroupExpandCollapseChangingEventArgs.html) of the `GroupCollapsing` event provides the information about the collapsing group and it contains the following members:
 
 [Groups](https://help.syncfusion.com/cr/xamarin/Syncfusion.ListView.XForms.GroupExpandCollapseChangingEventArgs.html#Syncfusion_ListView_XForms_GroupExpandCollapseChangingEventArgs_Groups): Gets a list of groups being collapsed.
-[Cancel](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?f1url=%3FappId%3DDev14IDEF1%26l%3DEN-US%26k%3Dk(System.ComponentModel.CancelEventArgs.Cancel)%26rd%3Dtrue&view=net-5.0): Decides whether to cancel the group collapsing or not.
+[Cancel](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?f1url=%253FappId%3DDev14IDEF1&l=EN-US&k=k(System.ComponentModel.CancelEventArgs.Cancel)&rd=true&view=net-5.0): Decides whether to cancel the group collapsing or not.
```
Comment (Yvone-Atieno) [inline Xamarin/ListView/Grouping.md:619]: Check this link

File: Xamarin/ListView/Grouping.md [hunk 3]
```diff
@@ -573,7 +573,7 @@ The [SfListView.GroupExpanding](https://help.syncfusion.com/cr/xamarin/Syncfusio
 The [GroupExpandCollapseChangingEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.ListView.XForms.GroupExpandCollapseChangingEventArgs.html) of the `GroupExpanding` event provides the information about the expanding group and it has the following members:
 
 [Groups](https://help.syncfusion.com/cr/xamarin/Syncfusion.ListView.XForms.GroupExpandCollapseChangingEventArgs.html#Syncfusion_ListView_XForms_GroupExpandCollapseChangingEventArgs_Groups): Gets a list of groups being expanded.
-[Cancel](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?f1url=%3FappId%3DDev14IDEF1%26l%3DEN-US%26k%3Dk(System.ComponentModel.CancelEventArgs.Cancel)%26rd%3Dtrue&view=net-5.0): Decides whether to cancel the group expansion or not.
+[Cancel](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?f1url=%253FappId%3DDev14IDEF1&l=EN-US&k=k(System.ComponentModel.CancelEventArgs.Cancel)&rd=true&view=net-5.0): Decides whether to cancel the group expansion or not.
```
Comment (Yvone-Atieno) [inline Xamarin/ListView/Grouping.md:576]: Check this link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/ListView/Grouping.md (hunk 2 above)
Comment (Edithabich) [reply Xamarin/ListView/Grouping.md:619]: fixed

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**